blueprints/kind-example/kind-example.yaml

```diff
@@ -26,7 +19,7 @@ spec:
               type: NodePort
     addons:
       - name: example-server
-        kind: chart
+        kind: HelmAddon
```

Comment:
The correct value is actually `chart`: https://github.com/MirantisContainers/boundless-operator/blob/main/controllers/addon_controller.go#L33
```
const (
	kindManifest            = "manifest"
	kindChart               = "chart"
	addonHelmchartFinalizer = "boundless.mirantis.com/helmchart-finalizer"
	addonManifestFinalizer  = "boundless.mirantis.com/manifest-finalizer"
	addonIndexName          = "helmchartIndex"
	helmJobNameTemplate     = "helm-install-%s"
)
```